src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-list/pool-list.component.html

```diff
@@ -0,0 +1,20 @@
+<nav aria-label="breadcrumb">
+  <ol class="breadcrumb">
+    <li class="breadcrumb-item">Cluster</li>
```

Comment:
You should remove this item from the breadcrumb since `Pools` is not a sub-menu of `Cluster`.